doc/en/Common.md

```diff
@@ -0,0 +1,101 @@
+# Overview
+
+Alchemist is an tool for assisting component-oriented FPGA development.
+It builds a digital circuit combining circuit components which are written in software programming language (i.e. C-language).
+The components are synthesized into HDL (Hardware Description Language) and synthezied, placed and routed onto FPGA using FPGA tools.
+
+# Environment
+
+- OS: Ubuntu 18.04 LTS
+- Python 3.7>=
+- FPGA tools: Vivado, Vivado HLS (2019.2)
+- Fast RTPS (v1.7.2)
+
+# Installation
+
+## Fast RTPS
+
+### Dependency
+
+- g++, make
+- CMake
+- Git
+- JDK8
+- Gradle
+
+```
+$ sudo apt install build-essential cmake git openjdk-8-jdk gradle -y
+$ sudo apt install libncurses5 -y
```

Comment:
Thank you for your request!

Please let me confirm about libncurses5.
Is this required to run alchemist, Vivado or VivadoHLS?
I'm not sure if libncurses5 is required while using alchemist.

Reply:
[libncurses5 is required by Vivado for simulation.](https://www.xilinx.com/support/answers/63794.html) However, Fast-RTPS does not require it. 

Therefore, I will remove the line.

Reply:
Thanks!